mon-pix/app/components/account-recovery/update-sco-record-form.gjs

```diff
@@ -102,11 +102,10 @@ export default class UpdateScoRecordFormComponent extends Component {
 
     {{#if this.isNewAccountRecoveryEnabled}}
 
-      <div class="account-recovery__content--form-quit-button-container">
-        <PixButtonLink @route="logout" class="account-recovery__content--actions update-sco-record-form__buttons">
-          {{t "common.actions.quit"}}
-        </PixButtonLink>
-      </div>
+      <PixButtonLink @route="logout" class="account-recovery__content--actions update-sco-record-form__buttons">
+        {{t "common.actions.quit"}}
+      </PixButtonLink>
+
```

Comment:
Pourquoi ces lignes vides à l’intérieur du `{{#if this.isNewAccountRecoveryEnabled}}` ? 

Je suggère d'enlever les lignes vides autour du `<PixButtonLink>…</PixButtonLink>`.

```suggestion
      <PixButtonLink @route="logout" class="account-recovery__content--actions update-sco-record-form__buttons">
        {{t "common.actions.quit"}}
      </PixButtonLink>
```